### PR TITLE
Tutorials for Hello World and Python data management

### PR DESCRIPTION
**Pull request by rfbaumgartner**
- link from block documentation to examples using the block
- use graphviz for explaining the data flow
- tutorial for Hello World
- tutorial for using Python for data management

related to https://github.com/inseri-swiss/inseri-swiss/issues/79
covering also covering https://github.com/inseri-swiss/inseri-swiss/issues/84

### CODE REVIEW

File: docs/tutorials/redirect-fields-with-python.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+author = data["author"]
+year = data["year"]
+publisher = data["publisher"]
+```
+
+Obviously, this is not yet complete because `data` is not defined anywhere in the code.
+
+## Step 4: Open the extended view for the python block
+
+Under "Inputs from Blocks", enter the variable name `data` and accept ith with selecting the `+`.
```
Comment (aplamada) [inline docs/tutorials/redirect-fields-with-python.md:48]: `ith` - typo, it should be `it`

File: docs/tutorials/redirect-fields-with-python.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+You can now select `webApi-book: data` to connect this block to the web API block.
+
+Now, you can set several output channels.
+As with the input, enter a variable name from the script into the textbox and accept with `+`.
+For `title` and `author` set the content type to "Text".
```
Comment (aplamada) [inline docs/tutorials/redirect-fields-with-python.md:53]: what about using json also for `title` and `author`. Not the most obvious choice but one can argue that is one of the most used content types so to get experience with it :D.

File: docs/tutorials/redirect-fields-with-python.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+Now, add three text users.
+Set their inputs to `python-divide: author`, `python-divide: year`, `python-divide: title`, `python-divide: publisher`, respectively.
+
+## Step 6: Save and publish
+
+You can save and publish your post like any WordPress post.
+
+Once published (or in preview), you will see the text editor with the content as described above and a dropdown.
+The images will load after selecting from the dropdown.
```
Comment (aplamada) [inline docs/tutorials/redirect-fields-with-python.md:73]: add the object if needed: "The images will load after selecting **it** ..."

**Review by aplamada — COMMENTED** [review]
Great work. See my little comments in the review.

File: docs/tutorials/redirect-fields-with-python.md (hunk 2 above)
Comment (rfbaumgartner) [reply docs/tutorials/redirect-fields-with-python.md:53]: I wanted to have something else than only json but maybe it's a bit too constructed this way. An option would be to re-write it to have one value in json and the other as text but the best would be an example where it matters.

File: docs/tutorials/redirect-fields-with-python.md (hunk 3 above)
Comment (rfbaumgartner) [reply docs/tutorials/redirect-fields-with-python.md:73]: Oops, there is some content that's been copied from _hello world_ and not upated.

File: docs/tutorials/redirect-fields-with-python.md (hunk 2 above)
Comment (aplamada) [reply docs/tutorials/redirect-fields-with-python.md:53]: I don't have a strong opinion, it was only a remark.

**Pull request merged**